src/anamoly_detection/anomaly_engine.py

```diff
@@ -0,0 +1,76 @@
+from scapy.all import Packet
+from src.anamoly_detection.equation_parser import parse_equation
+from src.database.models import AnomalyEquations
+from src.anamoly_detection.frequency_signatures.traffic.traffic_layer_frequency_signature \
+    import TrafficLayerFrequencySignature
+
+
+class AnomalyEngine:
+    """
+    Engine object used to manage anomaly signatures
+    """
+    def __init__(self, connection, frequency_signatures=[], traffic_signatures=[]):
```

Comment:
Should the signature engines ever be null on initialization?

Reply:
Not sure I follow, could you expand upon this?